templates/vue/package.json

```diff
@@ -19,12 +19,14 @@
     "babel-polyfill": "^6.26.0",
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.13.1",
+    "jquery": "^1.9.1",
```

Comment:
Any reason why this was installed? We don't use jQuery in the vue build.

Reply:
Will remove